atlasdb-cassandra-integration-tests/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/AbstractCassandraKeyValueServiceIntegrationTest.java

```diff
@@ -251,9 +253,10 @@ private static String getInternalTestTableName() {
     }
 
     @Test
-    @SuppressWarnings("Slf4jConstantLogMessage")
+    @SuppressWarnings("CompileTimeConstant")
     public void shouldNotErrorForTimestampTableWhenCreatingCassandraKvs() {
-        verify(logger, never()).error(startsWith("Found a table {} that did not have persisted"), anyString());
```

Comment:
funnily enough this worked because LoggingArgs.tableRef(...), which is logged in C*KVSImpl, is not a String ☠️